src/Bicep.Core/Workspaces/BicepFile.cs

```diff
@@ -33,5 +38,7 @@ public BicepFile(BicepFile original)
         public ProgramSyntax ProgramSyntax { get; }
 
         public SyntaxHierarchy Hierarchy { get; }
+
+        public ImmutableDictionary<int, DisableNextLineDirectiveEndPositionAndCodes> DisableNextLineDiagnosticDirectivesCache { get; }
```

Comment:
Rather than storing the dictionary itself (which defeats the benefit of using `Lazy<T>` by forcing calculation), why not just store a reference to the `DisabledDiagnosticsCache` instance?

Reply:
Sure, updated here:
https://github.com/Azure/bicep/commit/a49bf6e9a4703dc193b93b3e15f745863f9d8ce2